build.sh

```diff
@@ -2,7 +2,17 @@
 
 set -euo pipefail
 
-url="${1:-}"
+mode="development"
```

Comment:
@MeganSteenkamp I have defaulted to development here - what do you think?

Reply:
I prefer not to override their default of `production` mode but happy either way :)

Reply:
@AndyClifford I think this is ok 👍